src/components/Tag.svelte

```diff
@@ -1,9 +1,9 @@
 <script lang="ts">
   export let label: string;
-  export let size: "sm" | "md" | "lg" = "md";
+  export let size: "xs" | "sm" | "md" | "lg" = "md";
 </script>
 
-<span class={`tag tag--${label} tag--${size}`}>
+<span class={`tag tag--${label.toLowerCase()} tag--${size}`}>
```

Comment:
This should work as well, and it's slightly more readable imo: `class="tag tag--{label.toLowerCase()} tag--{size}"`

Reply:
Of course, thanks